fieldservice_stock/models/fsm_equipment.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class FSMEquipment(models.Model):
+    _inherit = 'fsm.equipment'
+    _description = "FSM Equipment Inherits"
+
+    lot_id = fields.Many2one(
+        'stock.production.lot', string='Lot/Serial #')
+    current_stock_location_id = fields.Many2one(
+        'stock.location', string='Current Inventory Location',
+        compute='_get_current_stock_loc_id')
+
+    @api.multi
+    def _get_current_stock_loc_id(self):
+        for equipment in self:
+            if equipment.lot_id and equipment.lot_id.product_id:
+                equipment.current_stock_location_id = \
+                    equipment.lot_id.product_id.property_stock_inventory \
```

Comment:
@smangukiya `current_location_location_id` should be set to the current location of the serial number: when the serial number is delivered to the customer location, the current location should change to `Partner Location / Customers`.